src/components/MovieItem.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <router-link :to="{ name: 'Movie', params: { movieId: movie.id } }">
```

Comment:
I cannot see such a route in your `router/index.js` file. Maybe you can add just an empty page for now?